### ISSUE TITLE
Support for CSparseMatrix

### ISSUE THREAD

**Issue by FFroehlich** [symengine/symengine.py#228]
I am currently working on code that computes matrix-matrix products of large, sparse matrices (>1k rows/columns, similar number of non-zero entries), which is relatively slow (multiple hours per product). I assume that this could be computed much faster by storing these matrices in a sparse representation and computing the products using respective sparse routines. However, I could not find any support for the sparse matrix class `CSRMatrix` in this wrapper.

Is this already implemented and I simply did not look in the right place? Are there any plans to support `CSRMatrix` in this wrapper in the near future?

**Comment by FFroehlich** [comment]
Do I correctly understand that  implementation would simply require replicating https://github.com/symengine/symengine.py/blob/5496ffcb4d9d24ee4af2754aafc124c5fc0e21c5/symengine/lib/symengine.pxd#L793 and adapting it to the CSRMatrix class?

**Comment by bjodah** [comment]
Yes, together with the corresponding adaptions in symengine_wrapper.pyx and adding a few tests to the test suite.

**Comment by laisnegautier** [comment]
Hi, will we see a bind for CSRMatrix soon?
